frontend/components/product-list/sections/FilterableProductsSection/FacetsAccordion.tsx

```diff
@@ -84,19 +87,31 @@ type FacetAccordionItemProps = AccordionItemProps & {
    refinedCount: number;
    productList: ProductList;
    isExpanded: boolean;
+   length: number;
 };
 
 export const FacetAccordionItem = forwardRef<FacetAccordionItemProps, 'div'>(
-   ({ attribute, refinedCount, productList, isExpanded, ...props }, ref) => {
+   (
+      { attribute, refinedCount, productList, isExpanded, length, ...props },
+      ref
+   ) => {
       const { items } = useFilteredRefinementList({ attribute });
       const { hits } = useHits();
       const isProductListEmpty = hits.length === 0;
       const hasApplicableRefinements = items.length > 0;
-      const isDisabled = isProductListEmpty || !hasApplicableRefinements;
+      let isDisabled = isProductListEmpty || !hasApplicableRefinements;
+      let isHidden = !hasApplicableRefinements && !isProductListEmpty;
 
-      const formattedFacetName = formatFacetName(attribute);
-      const isHidden = !hasApplicableRefinements && !isProductListEmpty;
+      if (isHidden && attribute !== 'worksin') {
```

Comment:
Why the special case for worksin here?

Reply:
Because then in the empty facet case, the only grayed out facet that was displayed was worksin.
I wasn't sure if that was okay so I added this condition to make sure worksin isn't the only one that is rendered.